paddle/fluid/framework/prune.cc

```diff
@@ -81,19 +83,47 @@ bool HasFalseTarget(const proto::OpDesc& op_desc) {
 }
 
 int GetSubBlockIndex(const proto::OpDesc& op_desc) {
+  // the block index >= 0, so -1 is used to indicate "NotFound".
```

Comment:
Better to capitalize the first word since it is a new sentence.

Reply:
Done, thanks.